unit/EspmTest.cpp

```diff
@@ -379,3 +380,40 @@ TEST_CASE("Correctly parses tree structure", "[espm]")
             0x4f838, 0x50015, 0x69857, 0x6ed38, 0x94b35, 0xc350d, 0xc97eb,
             0xd45f0, 0x104217 });
 }
+
+TEST_CASE("Testing values", "[espm]")
```

Comment:
Here single test case does 2 jobs:

1. do something with PartOne, Actor, etc. but actually, it doesn't test anything. it should test that `GetBaseActorValues` returns expected values instead.
2. test that espm is able to parse RACE record.

We need to separate these tests. I think 1-st goes to a separate file, 2-nd stays in EspmTest.cpp

Reply:
by the way, tests fail. something is wrong, launch and debug tests locally.